website/blog/2020-01-30-Own-Your-Data-Pilot.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Update #1 - Pilot project (Tally Zoo) with OwnYourData
+author: Dixon Siu
+authorURL: https://twitter.com/dixon_siu
+---
+
+During the winter vacation, [OwnYourData](https://www.ownyourdata.eu/)'s chief engineer had been working hard understanding Personium APIs and finally Tally Zoo [Documentation](http://bit.ly/TallyZoo_with_CEPS ) can access Personium user's data store (Box). We achieved the followings in current implementation:  
+
+1. Personium user authentication from TallyZoo  
+1. Retrieve Personium Cell access token to access the data store  
+1. Store and remove JSON files  
+
+Even though our goal (interoperability of personal data store) is still far away, [MyData](https://mydata.org) movement continues to drive collaborations among subject matter experts and passionate participants. Perseverance will finally lead us to our goal.  
+
+We will provide the following improvements for the next stage.  
+
+1. Add a launcher from Personium's HomeApp  
+1. Retrieve Personium Box access token to access the data store  
+1. Create/Delete OData table  
+1. Add/Delete OData table entry  
+
+<img src="https://dixonsiu.demo-jp.personium.io/MyData/images/ownyourdata.png" width="150px" >  
+Original news here: https://personium.io/blog/2019/12/09/Own-Your-Data-Pilot/  
+
+<!--truncate-->
+
+## Demo  
```

Comment:
```suggestion
## Demo  
Note:  You need a special configuration on your cell to perform the following procedure. 
```

Reply:
Should add some text like above to avoid misleading cell owners to try this operation now.